bindata/bootkube/config/bootstrap-config-overrides.yaml

```diff
@@ -24,4 +24,11 @@ storageConfig:
   certFile: /etc/kubernetes/secrets/etcd-client.crt # origin 3.11: master.etcd-client.crt
   keyFile: /etc/kubernetes/secrets/etcd-client.key # origin 3.11: master.etcd-client.key
   urls: {{range .EtcdServerURLs}}
-  - {{.}}{{end}}
\ No newline at end of file
+  - {{.}}{{end}}
+admissionPluginConfig:
+  openshift.io/RestrictedEndpointsAdmission:
+    configuration:
```

Comment:
is this completely missing in the defaultconfig.yaml? We cannot merge those sub-objects, only replace them.